101-aci-dynamicsnav/azuredeploy.json

```diff
@@ -0,0 +1,150 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {    
+      "name": {
+          "type": "string",
+          "metadata": {
+              "description": "Name for the container group"
+          },
+          "defaultValue": "acinavcontainergroup"
+      },    
+      "navRelease": {
+          "type": "string",
+          "metadata": {
+              "description": "After the colon: Identifier for the NAV release like 2017-cu11-de or devpreview-finus (always keep 'microsoft/dynamics-nav:')"
+          },
+          "defaultValue": "microsoft/dynamics-nav:2017"
+      },    
+      "username": {        
+          "type": "string",
+          "metadata": {
+              "description": "Username for your NAV super user"
+          },
+          "defaultValue": "admin"
+      },
+      "password": {
+          "type": "string",
+          "metadata": {
+              "description": "Password for your NAV super user and your sa user on the database"
+          },
+          "defaultValue": ""
+      },    
+      "cpuCores": {
+          "type": "string",
+          "metadata": {
+              "description": "The number of CPU cores to allocate to the container"
+          },
+          "defaultValue": "2.0"
+      },
+      "memoryInGb": {
+          "type": "string",
+          "metadata": {
+              "description": "The amount of memory to allocate to the container in gigabytes. Provide a minimum of 2 as he container will include SQL Server and NAV NST"
+          },
+          "defaultValue": "2.0"
+      },
+      "acceptEula": {
+          "type": "string",
+          "metadata": {
+              "description": "Change to 'Y' to accept the end user license agreement. This is necessary to successfully run the container"
```

Comment:
to what EULA is this agreeing to?

Reply:
Very good question :) The official images just ask you to accept the EULA but don't have a link or something. I'll find out

Reply:
Got feedback and added the link